gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -345,3 +351,35 @@ nnsconf_get_fullpath (const gchar * subpluginname, nnsconf_type type)
   g_free (filename);
   return ret;
 }
+
+/** @brief Public function defined in the header */
+const gint
+nnsconf_get_value (nnsconf_type_boolean type)
+{
+  int i;
+  gint ret = -1;
+
+  switch (type) {
+    case NNSCONF_VAL_TF_MEM_OPTMZ:
+      for (i = 0; i < CONF_SOURCES; i++) {
+        if (conf.valueTF_MEM_OPTMZ[i]) {
+          if (!g_strcmp0 ("1", conf.valueTF_MEM_OPTMZ[i]) ||
+              !g_strcmp0 ("TRUE", conf.valueTF_MEM_OPTMZ[i]) ||
+              !g_strcmp0 ("True", conf.valueTF_MEM_OPTMZ[i]) ||
+              !g_strcmp0 ("true", conf.valueTF_MEM_OPTMZ[i]))
```

Comment:
Use g_ascii_strncasecmp  ( https://developer.gnome.org/glib/stable/glib-String-Utility-Functions.html#g-ascii-strncasecmp ) instead of strcmp and forget about cases. You will need to handle NULL before calling it though.